src/analytics/Track.tsx

```diff
@@ -0,0 +1,40 @@
+import { PureComponent } from "react";
```

Comment:
Why is this here?

Reply:
Just migrated the original to TypeScript, but didn't move it to functional component. It's a class component that extends PureComponent, the old way.